Imagine/Filter/Loader/RotateFilterLoader.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Liip\ImagineBundle\Imagine\Filter\Loader;
+
+use Imagine\Image\ImageInterface;
+use Imagine\Image\ManipulatorInterface;
+
+/**
+ * Loader for Imagine's basic rotate method
+ *
+ * @author Bocharsky Victor <bocharsky.bw@gmail.com>
+ */
+class RotateFilterLoader implements LoaderInterface
+{
+    /**
+     * Loads and applies a filter on the given image.
+     *
+     * @param ImageInterface $image
+     * @param array $options
+     *
+     * @return ManipulatorInterface
+     */
+    public function load(ImageInterface $image, array $options = array())
+    {
+        $angle = isset($options['angle']) ? intval(round(floatval($options['angle']), 0)) : 0;
```

Comment:
In my case, I use this filter at runtime in my twig templates, and my angle evaluate from several params and can be `float`. But `Imagine\Image\ManipulatorInterface::rotate` method expect angle as `int`, so better to type cast to `int`.